src/Microsoft.Azure.AppConfiguration.IsolatedAzureFunctions/AzureAppConfigurationRefreshExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using Microsoft.Azure.Functions.Worker;
```

Comment:
This is the namespace and package of isolated Azure functions. I hear multiple names for this new mode. Isolated, out-of-process, and worker mode. I think the best bet will be to be consistent with their packaging scheme.
`Microsoft.Azure.AppConfiguration.Functions.Worker`

@zhenlan @avanigupta

Reply:
Avani and I also chatted about the mess of naming :). What you proposed sounds good to me.